banzai/tests/test_readnoise.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import mock
+import numpy as np
+
+from banzai.readnoise import ReadNoiseLoader
+from banzai.tests.utils import FakeContext, FakeLCOObservationFrame, FakeCCDData
+
+pytestmark = pytest.mark.read_noise
+
+
+@pytest.fixture(scope="module")
+def set_random_seed():
+    np.random.seed(81232385)
+
+
+def make_test_readnoise(nx, ny, input_readnoise, make_3d=False):
+    if make_3d:
+        final_shape = (4, ny, nx)
+    else:
+        final_shape = (ny, nx)
+    return np.random.normal(0.0, input_readnoise, final_shape)
+
+
+def test_null_input_images():
+    tester = ReadNoiseLoader(None)
+    image = tester.run(None)
+    assert image is None
+
+
+@mock.patch("banzai.lco.LCOFrameFactory.open")
+@mock.patch("banzai.calibrations.CalibrationUser.get_calibration_file_info", return_value={"filename": "test.fits"})
+def test_adds_good_noise_map(mock_noise_map_file_info, mock_noise_map, set_random_seed):
+    image = FakeLCOObservationFrame(hdu_list=[FakeCCDData(memmap=False)])
+    master_image = FakeLCOObservationFrame(
```

Comment:
nitpick, not even sure this is a convention we can change, but maybe change master to main?

Reply:
Done